### PR TITLE
Fix kubectl top sort-by cpu and sort-by memory options

### LINKED ISSUE #81270: `kubectl top --sort-by` is not sorting correctly
The --sort-by option to kubectl top was introduced from v1.15.2 and it accepts only 2 params i.e. cpu and memory.

e.g. 
kubectl top po --all-namespaces --sort-by=cpu
kubectl top po --all-namespaces --sort-by=memory

The output is not in a sorted manner.

**Environment**:
- Kubernetes version (use `kubectl version`): v1.15.2
- Cloud provider: GCP/GKE

### PR DESCRIPTION
**Pull request by tlereste**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

This PR fix the kubectl top **sort-by=cpu** and **sort-by=memory** options.
The sort-by option doesn't sort correctly the pods and nodes.

Two problems ares fixed in the PR :
- Fix the Swap functions to sort nodes and pods
- Use MilliValue() instead of Values() to compare CPU metrics

**Which issue(s) this PR fixes**:

Fixes #81270

**Special notes for your reviewer**:
This is my first PR, please be indulgent :)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by tlereste** [comment]
ping @liggitt 
This is my first PR but nobody review it. 
Did I forget something ?
Thanks !

**Comment by liggitt** [comment]
> This is my first PR but nobody review it.
> Did I forget something ?

apologies, v1.16 consumed a lot of attention over the last week. you should be all set, this just needs review by sig-cli

/ok-to-test
/unassign
/assign @soltysh 

@soltysh or @seans3, can you review or delegate?

**Comment by tlereste** [comment]
No problem, thanks ! :)

**Comment by tlereste** [comment]
ping @liggitt @soltysh 
My tests are successful, I am ready for the next step :)
Can you please assign to somebody for PR review ?
Thanks again !

**Comment by oke-py** [comment]
Any updates?

**Comment by tlereste** [comment]
No updates @oke-py :confused: !
I'm waiting for PR review by @soltysh , @seans3 , @liggitt or someone else ! :flushed:

**Comment by tlereste** [comment]
/assign deads2k
/assign neolit123
Hi @deads2k and @neolit123 could you please review this PR or delegate ?
My PR is blocked since 3 months and I look for a review before taking another issue !
Thanks to you in advance :pray: !

**Comment by neolit123** [comment]
/unassign
if you are not getting a review, try asking in the #sig-cli channel.

**Review by nikopen — APPROVED** [review]

File: staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go]: Why we need to go with `MilliValue` here? Are you aware this can overflow thus causing more errors?

**Review by soltysh — CHANGES_REQUESTED** [review]
/assign

File: staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go
Comment (tlereste) [reply staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go]: Because CPU metrics are always collected with `MilliValue` (instead of MEMORY metrics which are collected with `Value`)

For exemple, for one sample node, `MilliValue` will return _1628_ and `Value` will return _2_. (Value = MilliValue / 1000 round up)

Another exemple show you existing code at the end of _metrics_printer.go_ file : 

```
func printSingleResourceUsage(out io.Writer, resourceType v1.ResourceName, quantity resource.Quantity) {
	switch resourceType {
	case v1.ResourceCPU:
		fmt.Fprintf(out, "%vm", quantity.MilliValue())
	case v1.ResourceMemory:
		fmt.Fprintf(out, "%vMi", quantity.Value()/(1024*1024))
	default:
		fmt.Fprintf(out, "%v", quantity.Value())
	}
}

```

Comment (soltysh) [reply staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go]: Fair, thanks for the details.

**Review by soltysh — APPROVED** [review]
/approve
/priority backlog
Please squash your changes to a single commit.

**Review by nikopen — APPROVED** [review]

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**
